src/utils/ItemLocaleUtils.ts

```diff
@@ -0,0 +1,20 @@
+import {localizedLanguagesStrings} from '../i18n';
+import {CarouselDataItem} from '../screens/HomeScreen/components';
```

Comment:
Utils should not import from screens or components.

Reply:
Removed as the utils are changed